src/components/expression/expression.js

```diff
@@ -79,17 +80,25 @@ genotet.ExpressionView = function(viewName, params) {
 
   // Set up data loading callbacks.
   $(this.container).on('genotet.ready', function() {
+    this.data.tfaFileName = params.tfaFileName;
```

Comment:
You can write whatever you want to this.data.tfa. this.data.tfa does not need to be exactly the same as what was returned from the server.